content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst

```diff
@@ -2,60 +2,114 @@
+
+Configure the remaining fields in the :guilabel:`Additional Information` field as follows:
+
+- :guilabel:`Company`: the company whose warehouse the location is inside of. Leave this field blank
+  if this location is shared between companies.
 - :guilabel:`Is a Scrap Location?`: check this box to allow for scrapped/damaged goods to be stored
-  in this location
+  in this location.
 - :guilabel:`Is a Return Location?`: check this box to allow products to be returned to this
-  location
-- :guilabel:`Barcode`: the barcode number assigned to the location
-- :guilabel:`Removal Strategy`: the :doc:`strategy <../removal_strategies>` for how items
-  should be taken from inventory
+  location.
+- :guilabel:`Barcode`: the barcode number assigned to the location.
+- :guilabel:`Replenish Location`: check this box to get all quantities to replenish at this
+  location.
+
+In the :guilabel:`Logistics` section, in the :guilabel:`Removal Strategy` field, click the drop-down
+menu and select the :doc:`removal strategy <../removal_strategies>` for how items should be removed
+from this location.
 
-.. image:: use_locations/new-location-form.png
+In the :guilabel:`Cyclic Counting` section, change the value in the :guilabel:`Inventory Frequency
+(Days)` field from the default `0`, if necessary.
+
+.. image:: use_locations/use-locations-cyclic-counting.png
    :align: center
-   :alt: The form for creating a new location.
+   :alt: Cyclic Counting section of new location creation form.
+
+When different than `0`, inventory count date for products stored at this location are automatically
+set at the defined frequency.
 
 .. _inventory/location-hierarchy:
 
 Create location hierarchies
 ===========================
 
 The *Parent Location* setting on the new location form allows for a location to exist within a
-warehouse or another location. Every location can serve as a parent location, and every parent
-location can have multiple locations within it, allowing for the creation of a virtually infinite
-hierarchical structure.
+warehouse, or below another location.
+
+Every location can serve as a parent location, and every parent location can have multiple locations
+within it, allowing for the creation of a hierarchical structure.
 
 .. example::
-   Location hierarchy could be organized so that a shelf is located within an aisle, which is
-   located within a room, which is located within the overall warehouse.
+   Location hierarchy could be organized so that a small refrigerator is located on a specific
+   shelf, which is located in the warehouse.
+
+   .. image:: use_locations/use-locations-hierarchy.png
+     :align: center
+     :alt: Location hierarchy of small refrigerator, on a shelf in a warehouse.
+
+   To create the location hierarchy in the example above, set the warehouse stock as the parent of
+   the shelf, and the shelf as the parent of the small refrigerator. This can be adapted to a
+   hierarchy of any magnitude.
```

Comment:
I'd also spell out the resulting location hierarchy `WH/Stock/Shelf 2/Small Refrigerator` here, as well!

Reply:
Suggestion implemented ✅